basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
+
+echo $PATH_TO_FILES $PATH_TO_SHARD
+
+basin-generator $PATH_TO_FILES $PATH_TO_JSON
+basin $PATH_TO_JSON $PATH_TO_SHARD/output.shard
+eminem regenerate $PATH_TO_SHARD
```

Comment:
Maybe related my previous comment, but we don't need mix these two directories, since we will be able to use any custom shard with the custom-content option in e-k-l.